Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -160,25 +174,6 @@ class InAppMessageViewTest: UnitTest {
         XCTAssertTrue(webViewBeforeFetch === webViewAfterFetch)
     }
 
-    @MainActor
-    func test_givenAlreadyShowingInAppMessage_whenNewMessageFetched_expectDoNotReplaceContents() async {
```

Comment:
Removed this test because `test_expiration_givenMessageExpired_givenNewMessageFetched_expectDisplayNextMessage` is testing this logic for us already, making this test no longer valuable.